bridge/bridge/api/static/openapi/openapi.yaml

```
@@ -0,0 +1,919 @@
+openapi: 3.0.3
+info:
+  title: Bridge API Documentation
+  description: |
+    API for handling token wrapping and unwrapping between native and wrapped networks.
+    The API exposes the internal bridge state so external systems can monitor request processing,
+    payout status, errors, and fee/conversion estimates.
+    Unwrap operations are available only when the bridge is running in wrap or stake mode
+    (they are not available in swap mode).
+  version: 1.0.0
+servers:
+  - url: "../../"
+    description: Current server
+  - url: "http://localhost:5000"
+    description: Local development server
+tags:
+  - name: Bridge
+    description: Provides access to bridge configuration.
+  - name: Wrap
+    description: Endpoints related to wrapping tokens from the native chain into the wrapped chain.
+  - name: Unwrap
+    description: |
+      Endpoints related to unwrapping tokens from the wrapped chain back into the native chain.
+      Unwrap operations are supported only in wrap and stake bridge modes (please check the bridge README for more details).
+
+paths:
+  /:
+    get:
+      summary: Returns bridge configuration.
+      tags:
+        - Bridge
+      operationId: getBridgeStatus
+      responses:
+        200:
+          description: Detailed bridge configuration.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BridgeConfiguration'
+        500:
+          description: Unexpected internal error.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+
+  /wrap/requests:
+    get:
+      summary: Returns wrap requests information.
+      description: |
+        Retrieves a paginated list of wrap requests detected on the native network.
+        These represent user-initiated deposits that trigger wrapping.
```

```suggestion
        Retrieves a paginated list of the ongoing wrap requests detected on the native network.
        These represent user-initiated deposits that trigger wrapping.
        Once complete, the requests disappear from this list.
```
Am I correct? The list only includes requests currently being processed?

Things that might be useful to mention here too:
- How can I recover a request that has already been completed, either failed or succeeded?
- Which endpoint do I use to make such a request?
- Is there any other way to monitor requests, like websockets?

In general, it's always good to include lots of links to other parts of the documentation, so if this section is not exactly what the reader was looking for, at least they have a thread to pull and continue looking. Otherwise, they have no choice but to read the whole spec.

---

API returns all requests / errors. Not sure if it's worth to mention. I provided some changes to specification, hopefully it better now.

---

Added that informaion